derek/ner/nn/classifier.py

```diff
@@ -33,15 +33,16 @@ def __init__(self, graph, feature_extractor: NERFeatureExtractor, feature_comput
         self.saver = saver
         self.post_processor = post_processor
 
-    def predict_doc(self, doc: Document) -> List[Entity]:
+    def predict_doc_with_scores(self, doc: Document) -> Tuple[List[Entity], List]:
```

Comment:
Because this method is public its typing should be detailed.
```suggestion
    def predict_doc_with_scores(self, doc: Document) -> Tuple[List[Entity], List[float]]:
```